### ISSUE TITLE
why objects instead of arrays?

### ISSUE THREAD

**Issue by nbriz** [hbrown1994/10print#1]
https://github.com/hbrown1994/10print/blob/41b23e88176fe9e0a8c27e2e9b334fa6bdd01eb5/10print.js#L71-L80
why are u using an object here instead of just making it an array? same for the `CHARS` object on line 83. an array is essentially an object indexed by numbers (instead of by key strings), so if u have an object like this: `let x = { '0': ['a','b','c'] }` which u'd then reference like `x['0']` u could do something like `let x = [ ['a','b','c'] ]` && reference like `x[0]` ...which is less code. u're already using multidimetional arrays (which is what we call array's w/other arrays inside)... so yea, just curious, is there a reason for using objects in this case?

**Comment by hbrown1994** [comment]
No, I didn't have a reason. This is just some hacked together code I pulled from various Stack Overflow searches I was doing to try and get my weighted randomness function to work. I'll try and re-implement this as multidimensional arrays! Thanks
